efficient_full_table_scan_example_code/efficient_full_table_scan.go

```diff
@@ -18,7 +18,7 @@ const (
 	defaultNumberOfNodesInCluster     = 3
 	defaultNumberOfCoresInNode        = 8
 	defaultSmudgeFactor               = 3
-	defaultQueryTemplate              = "SELECT token(key) FROM keyspace1.standard1 WHERE token(key) >= %d AND token(key) <= %d;"
+	defaultQueryTemplate              = "SELECT token(key) FROM keyspace1.standard1 WHERE token(key) >= %s AND token(key) <= %s;"
 	defaultSelectStatementsOutputFile = "/tmp/select_statements.txt"
```

Comment:
Why %s? just write `token(key) >= ? AND token(key) <= ?` (and s/Template//).

Reply:
because (in the current code), we print the queries to a file.

Reply:
But it will print the bind markers, not the values.

Reply:
Oh, it won't.

Reply:
I guess you figured it out, but indeed, my first instinct was just to change the statement to `?`. But to keep the ability to print the statements, I replaced `%d` with `%s`, and then to generate the one prepared statement, I replace `%s` with `?`.

then for each query I use the base string to replace %s with the token themselves -> and that is what goes into the file.

The prepared string gets `Bind` called on it and that is what is used to actually generate the statement.

Reply:
@avikivity anything else here ?